product_combination_exclude/models/product_product.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Graeme Gellatly
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class ProductProduct(models.Model):
+
+    _inherit = 'product.product'
+
+    @api.multi
+    def create(self, vals):
+        vals = dict(vals or {})
```

Comment:
@lasley, please calm down your tone. Everyone is free to think in a different manner in the way to contribute, so if he thinks this is too much to ask for, I have already offer to help him in the PR for these things. If not, you can continue his work if you want to.